frontend/src/app/models/connectors/EmployeeApiConnector.ts

```diff
@@ -10,9 +10,8 @@ export class EmployeeApiConnector extends AbstractApiConnector {
         return this.http.get(this.apiRoute) as Observable<User>;
     }
 
-    public changePassword(id: number, firstPassword: String, secondPassword: String): Observable<User> {
-        console.log("password: " + firstPassword);
-        return this.http.put(this.apiRoute + "/changepassw/" + id, { firstPassword, secondPassword }) as Observable<User>;
+    public changePassw(firstPassword: string, secondPassword: string, oldPassword: string): Observable<User> {
+        return this.http.post(this.apiRoute + "/changepassw", {firstPassword: firstPassword, secondPassword: secondPassword, oldPassword: oldPassword}, {responseType: 'json'}) as Observable<User>;
```

Comment:
így lehetne egyszerüsíteni + a responseType direkt lett külön megadva?
```suggestion
        return this.http.post(this.apiRoute + "/changepassw", {firstPassword, secondPassword, oldPassword}) as Observable<User>;
```

Reply:
a responseType csak kísérletezés miatt lett külön megadva, kiszedtem, ahogy a kulcsokat is a body-ból